src/embed/app.ts

```diff
@@ -579,6 +587,7 @@ export class AppEmbed extends V1Embed {
         params[Param.ModularHomeExperienceEnabled] = modularHomeExperience;
         params[Param.CollapseSearchBarInitially] = collapseSearchBarInitially || collapseSearchBar;
         params[Param.EnableCustomColumnGroups] = enableCustomColumnGroups;
+        params[Param.EnableFlipTooltipToContextMenu] = enableFlipTooltipToContextMenu;
```

Comment:
Why are you adding this flag by default in iframe URL?
Also, is this flag specific to appEmbed only?

Reply:
i will remove default addition

And yes it is needed in other embed also will add it there also thanks